### PR TITLE
Slider: Fix tooltip issues and overlap

### PR DESCRIPTION
**Pull request by gabrielschulhof**
Revert commit causing mini slider misalignment and some tooltip inconsistencies.

This PR is really the same as #7992, but using the upstream repo.

### CODE REVIEW

**Comment by gabrielschulhof** [comment]
@jaspermdegroot @arschmitz said that we need to fix #6795 the right way before this can land. Wanna try that clearfix solution you were talking about in https://github.com/jquery/jquery-mobile/issues/6795#issuecomment-72270943 on this branch?

**Comment by gabrielschulhof** [comment]
I've tried some clearfix solutions: https://github.com/jquery/jquery-mobile/issues/6795#issuecomment-82308636 - if we can agree on one, I'd be happy to implement it.

**Comment by jaspermdegroot** [comment]
@gabrielschulhof 

I just commented on #6795.

Your first commit (the revert) does not only fix #7991 but also #7821. I think we should close one of those as duplicate. I am going to close PR #7825 in favor of this PR. See also my https://github.com/jquery/jquery-mobile/pull/7825#issuecomment-72269269 on that PR.

**Comment by gabrielschulhof** [comment]
I tested the clearfix and the slider non-overlap on pretty much everything browserstack had to offer, except Windows Phone. For some reason it couldn't load the page.

**Comment by gabrielschulhof** [comment]
http://jsbin.com/zuburu/1/

**Comment by jaspermdegroot** [comment]
I just tested on Windows Phone. Works well.

**Comment by gabrielschulhof** [comment]
Awesome!

File: tests/css/slider/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+			[ "widgets/forms/slider" ],
+			[ "init" ],
+			[ "slider_core.js" ]
+		]);
+	</script>
+
+	<link rel="stylesheet"  href="../../../css/themes/default/jquery.mobile.css"/>
+	<link rel="stylesheet" href="../../../external/qunit/qunit.css"/>
+
+	<script src="../../swarminject.js"></script>
```
Comment (arschmitz) [inline tests/css/slider/index.html]: We don't use test swarm why is this here?

**Comment by arschmitz** [comment]
:+1:

File: tests/css/slider/index.html (hunk 1 above)
Comment (gabrielschulhof) [inline tests/css/slider/index.html]: All our tests have that. I'll remove it.

Comment (arschmitz) [inline tests/css/slider/index.html]: Hmm guess i just never noticed it before. remove this one and we can remove the rest in a separate pr.
